src/puppetlabs/trapperkeeper/services/webserver/jetty9_service.clj

```diff
@@ -26,23 +33,38 @@
 (defservice jetty9-service
   "Provides a Jetty 9 web server as a service"
   WebserverService
-  [[:ConfigService get-in-config]]
+  {:required [ConfigService]
+   :optional [FilesystemWatchService]}
```

Comment:
I made this an `optional` dependency rather than a required one in order to not break any applications that don't already bootstrap in the tk-filesystem-watcher service on upgrading to the new Jetty version.  This also provides a meager knob for turning off the filesystem CRL watching if it proves to be problematic for some reason.  Longer term, though, it might make sense to just make this be required / always on - maybe in a future tk-jetty9 major release?